index.js

```diff
@@ -1,9 +1,10 @@
+import 'babel-polyfill';
```

Comment:
Just curiosity, what is `babel-polyfill` needed for and how is it used?

Reply:
`babel-polyfill` transpiles ES6+ language features to ES5 at runtime. This means you can run an ES6+ script with `node index.js` rather than building + `node dist/index.js` or `babel-node index.js`. Indeed, `babel-node` probably uses `babel-polyfill` internally.
 
Since we use `package.json:script:serve: node dist/index.js`, which implies a build step, we shouldn't also need the polyfill.

Reply:
Alas, `babel-polyfill` is necessary here. It includes the polyfills for async/await that are needed.

I don't understand it very deeply, but something along these lines is occurring:
- `babel` turns the JS into ES5
- neither async/await nor the features that async/await is sugar for (promises and generators) exist in ES5
- So the output code needs a polyfill for those features, and tries to use them
- But babel doesn't include polyfills for those features by default because thats extra bloat for most apps
- So we need to explicitly include it here.

(someone who knows more than me feel free correct anything above that is wrong :))

Specifically, because I started using async/await here, the output of babel tries to use a symbol `regeneratorRuntime` which doesn't exist. `regeneratorRuntime` is defined in `babel-polyfill`.

It's because we're targeting a pre-async/await/promises/generators version of javascript that the babel output needs a polyfill. We should probably just target a newer version of javascript.